bughouse_webserver/src/history.rs

```diff
@@ -53,7 +53,13 @@ fn get_timestamp_for_plotly(stats: &RawStats) -> Option<String> {
         .ok()
 }
 
-pub fn players_rating_graph_html(stats: &GroupStats<Vec<RawStats>>) -> String {
+#[derive(Clone, Copy)]
```

Comment:
IMHO it makes sense to also derive Debug on all types. You never know when you'll need it :)